Package.swift

```diff
@@ -18,6 +18,12 @@ var globalConcurrencyFlags: [String] = []
 // ])
 // #endif
 
+#if swift(>=5.6)
+var swiftSyntaxDep="SwiftSyntaxParser"
+#else
+var swiftSyntaxDep="SwiftSyntax"
+#endif
```

Comment:
- formatting is off needs space between the name and `=`
- please declare those as `let`, we never mutate them

Reply:
I went ahead and fixed this since otherwise the patch is LGTM  -- thank you very much!